code/src/components/Task.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { todos } from '../reducers/todos'
+import { DeleteX } from '../assets/DeleteX'
+import { CustomCheckbox } from 'components/CustomCheckbox'
+import styled from 'styled-components'
+
+export const Task = ({ itemIndex }) => {
+  //om item ändras så ändras hela
+  const item = useSelector(store => store.todos.list.items[itemIndex])
+
+  const dispatch = useDispatch()
+
+  //Remove task
+  const onRemoveClicked = event => {
+    dispatch(
+      todos.actions.removeTask({
+        itemIndex: itemIndex,
+      }
+      )
+    )
+  }
+
+  // done or not status
+  const handleOnChange = event => {
+    dispatch(
+      todos.actions.setDone({
+        itemIndex: itemIndex,
+        done: !item.done
+      }))
+  }
```

Comment:
It would be better to use a unique id instead of the index to deal with the done state (then you can use that as the key prop as well, when you're mapping over the array). There's an issue now that happens if you write some todos, check the first one and then remove it, the second one will "jump up" to index 0 and become checked.